code/modules/clothing/suits/jobs.dm

```diff
@@ -236,3 +236,18 @@
 	icon_state = "jamrock_blazer"
 	inhand_icon_state = "jamrock_blazer"
 
+//This kind of shenanigans exist because I have no other idea how to make those disco suits not togglable.
+/obj/item/clothing/suit/det_suit
+	name = "old trenchcoat"
+	desc = "Looks like mapper fucked up and placed wrong trenchcoat."
```

Comment:
Why does this type exist if you're replacing it?

Reply:
I don't want to touch Disco Elysium coats, they're too cool to have buttons, so I had to leave this so those can still exist without buttons.

Reply:
then replace this type with disco elysium coat

Reply:
> then replace this type with disco elysium coat

hmmmm

Reply:
Done and fixed with the last commit. @Fikou @ZeWaka